src/clients/callbackClient.ts

```diff
@@ -7,11 +7,11 @@ import { ICallbackData, ICallbackExportData, IConfig } from '../common/interface
 @singleton()
 export class CallbackClient extends HttpClient {
   public constructor(@inject(SERVICES.LOGGER) logger: Logger, @inject(SERVICES.CONFIG) private readonly config: IConfig) {
-    super(logger, '', 'requestCallback', config.get<IHttpRetryConfig>('httpRetry'));
+    super(logger, '', 'requestCallback', config.get<IHttpRetryConfig>('httpRetry'), false);
```

Comment:
should it be const and configurale params?

Reply:
fixed as requested